src/Database/Sql.php

```diff
@@ -797,8 +811,7 @@ public function update(array $params = []): array
      * @param string $table
      * @param string $column
      * @return bool
-     *
-     * @throws \Kirby\Exception\InvalidArgumentException If the column is invalid
+     * @throws \Kirby\Exception\InvalidArgumentException|\Throwable If the column is invalid
```

Comment:
Where does `Throwable` come from? I only see `InvalidArgumentException` in the method.

Reply:
As there are interlinked methods, you can follow the sequence below for `validateColumn` method:

> public function validateColumn() > validateColumn() > validateTable() > query() > hit()